src/index.js

```diff
@@ -166,12 +167,21 @@ const unexpectedEnzyme = {
     });
 
     childExpect.exportAssertion(
-      '<EnzymeWrapper> [not] to be checked',
+      '<ReactWrapper> [not] to be checked',
       (expect, reactWrapper) => {
         expect(reactWrapper.instance().checked, '[not] to be true');
       }
     );
 
+    childExpect.exportAssertion(
+      '<ShallowWrapper> [not] to be checked',
+      (expect, shallowWrapper) => {
+        const element = $(shallowWrapper.html());
```

Comment:
@sunesimonsen what do you think about using `cheerio`? I'm taking a similar approach as https://github.com/producthunt/chai-enzyme/blob/master/src/ShallowTestWrapper.js#L80

It seems that it's a common practise when using enzyme, as it has built-in support http://airbnb.io/enzyme/docs/api/render.html

Reply:
I sounds slow, so I think we should only do it in cases where the normal approach doesn't work.